infer/annotations/Makefile

```diff
@@ -22,12 +23,12 @@ all: $(ANNOTATIONS_JAR) $(PROCESSOR_JAR)
 
 $(ANNOTATIONS_JAR): $(ANNOT_SOURCES)
 	mkdir -p $(ANNOT_CLASSES)
-	javac -cp $(JSR_JAR) $(ANNOT_SOURCES) -d $(ANNOT_CLASSES)
+	javac -source $(TARGET_JDK_VERSION) -target $(TARGET_JDK_VERSION) -cp $(JSR_JAR) $(ANNOT_SOURCES) -d $(ANNOT_CLASSES)
 	cd $(ANNOT_CLASSES) && jar cvf $(ANNOTATIONS_JAR) com
 
 $(PROCESSOR_JAR): $(PROCESSOR_SOURCES)
 	mkdir -p $(PROCESSOR_CLASSES)
-	javac $(PROCESSOR_SOURCES) -cp .:$(JAVA_HOME)/lib/tools.jar -d $(PROCESSOR_CLASSES)
+	javac -source $(TARGET_JDK_VERSION) -target $(TARGET_JDK_VERSION) $(PROCESSOR_SOURCES) -cp .:$(JAVA_HOME)/lib/tools.jar -d $(PROCESSOR_CLASSES)
```

Comment:
I don't think this is needed since the annotation processor does not get DEX'd, I can remove it.